### ISSUE TITLE
Home Assistant WebSocket API Timeout

### ISSUE THREAD

**Issue by Stefano201** [home-assistant/supervisor#4889]
### Describe the issue you are experiencing

Hi, i have some issues with supervisor several times in a day with slowdowns and core reboots.

If I kill all my add-on I don't have this issue.
Any solutions?

### What type of installation are you running?

Home Assistant OS

### Which operating system are you running on?

Home Assistant Operating System

### Steps to reproduce the issue

1. launch node-red
2. launch mosquitto broker

### Anything in the Supervisor logs that might be useful for us?

```txt
This is the last log:


24-02-12 22:04:25 WARNING (MainThread) [supervisor.misc.tasks] Watchdog missed an Home Assistant Core API response.
24-02-12 22:09:44 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
24-02-12 22:16:11 ERROR (MainThread) [supervisor.homeassistant.api] Timeout on call http://172.30.32.1:8123/api/core/state.
24-02-12 22:16:16 INFO (MainThread) [supervisor.api.proxy] Home Assistant WebSocket API error: Cannot proxy websocket message of unsupported type: 257
24-02-12 22:16:16 INFO (MainThread) [supervisor.api.proxy] Home Assistant WebSocket API error: Cannot proxy websocket message of unsupported type: 257
24-02-12 22:16:17 INFO (MainThread) [supervisor.api.proxy] Home Assistant WebSocket API connection is closed
24-02-12 22:16:23 INFO (MainThread) [supervisor.api.proxy] Home Assistant WebSocket API connection is closed
24-02-12 22:17:01 ERROR (MainThread) [supervisor.homeassistant.api] Timeout on call http://172.30.32.1:8123/api/core/state.
24-02-12 22:17:02 WARNING (MainThread) [supervisor.misc.tasks] Watchdog missed an Home Assistant Core API response.
24-02-12 22:17:15 ERROR (MainThread) [supervisor.homeassistant.api] Timeout on call http://172.30.32.1:8123/api/core/state.
24-02-12 22:17:15 ERROR (MainThread) [supervisor.homeassistant.api] Timeout on call http://172.30.32.1:8123/api/core/state.
24-02-12 22:17:38 ERROR (MainThread) [supervisor.homeassistant.api] Timeout on call http://172.30.32.1:8123/api/core/state.
24-02-12 22:17:57 ERROR (MainThread) [supervisor.homeassistant.api] Timeout on call http://172.30.32.1:8123/api/core/state.
24-02-12 22:17:58 ERROR (MainThread) [supervisor.homeassistant.api] Timeout on call http://172.30.32.1:8123/api/core/state.
24-02-12 22:18:43 ERROR (MainThread) [supervisor.homeassistant.api] Timeout on call http://172.30.32.1:8123/api/core/state.
24-02-12 22:18:44 ERROR (MainThread) [supervisor.homeassistant.api] Timeout on call http://172.30.32.1:8123/api/core/state.
24-02-12 22:18:49 INFO (MainThread) [supervisor.api.proxy] Home Assistant WebSocket API request initialize
24-02-12 22:18:51 INFO (MainThread) [supervisor.api.proxy] WebSocket access from a0d7b954_nodered
24-02-12 22:18:51 INFO (MainThread) [supervisor.api.proxy] Home Assistant WebSocket API request initialize
24-02-12 22:18:51 INFO (MainThread) [supervisor.api.proxy] WebSocket access from a0d7b954_nodered
24-02-12 22:18:52 INFO (MainThread) [supervisor.api.proxy] Home Assistant WebSocket API request running
24-02-12 22:18:52 INFO (MainThread) [supervisor.api.proxy] Home Assistant WebSocket API request running
```
```


### System Health information

## System Information

version | core-2024.2.1
-- | --
installation_type | Home Assistant OS
dev | false
hassio | true
docker | true
user | root
virtualenv | false
python_version | 3.12.1
os_name | Linux
os_version | 6.1.63-haos-raspi
arch | aarch64
timezone | Europe/Rome
config_dir | /config

<details><summary>Home Assistant Community Store</summary>

GitHub API | ok
-- | --
GitHub Content | ok
GitHub Web | ok
GitHub API Calls Remaining | 5000
Installed Version | 1.34.0
Stage | running
Available Repositories | 1401
Downloaded Repositories | 24
HACS Data | ok

</details>

<details><summary>Home Assistant Cloud</summary>

logged_in | false
-- | --
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | ok

</details>

<details><summary>Home Assistant Supervisor</summary>

host_os | Home Assistant OS 11.5
-- | --
update_channel | stable
supervisor_version | supervisor-2024.01.1
agent_version | 1.6.0
docker_version | 24.0.7
disk_total | 57.8 GB
disk_used | 7.2 GB
healthy | true
supported | true
board | rpi4-64
supervisor_api | ok
version_api | ok
installed_addons | File editor (5.8.0), Advanced SSH & Web Terminal (17.1.0), Samba share (12.3.0), Grott Beta branch (2.8) (0.1.5), ialarm-mqtt (0.12.0), Node-RED (17.0.7), Mosquitto broker (6.4.0), Cloudflared (5.1.3)

</details>

<details><summary>Dashboards</summary>

dashboards | 1
-- | --
resources | 12
views | 3
mode | storage

</details>

<details><summary>Recorder</summary>

oldest_recorder_run | 10 febbraio 2024 alle ore 15:01
-- | --
current_recorder_run | 12 febbraio 2024 alle ore 22:36
estimated_db_size | 60.49 MiB
database_engine | sqlite
database_version | 3.44.2

</details>

### Supervisor diagnostics

_No response_

### Additional information

Core 2024.2.1
Supervisor 2024.01.1
Operating System 11.5
Frontend 20240207.1

**Comment by agners** [comment]
> If I kill all my add-on I don't have this issue.

Which add-on specifically?

It seems that the Core is not responding to API calls, presumably because it is overloaded doing work.

**Comment by Stefano201** [comment]
From yesterday I'm using Mosquito broker, Grott and Cloudflare add-ons only... without API issues

**Comment by Stefano201** [comment]
up

**Comment by Stefano201** [comment]
Thank u for your reply... But how can I downgrade the core? ha core update --version xxxx.xx.xx?

**Comment by Stefano201** [comment]
@agners I try to downgrade Core to 2023.12.0, everything ok...but I think I need to downgrade supervisor too.
My current version is 2024.01.1...but I don't know which version was release with 2023.12.0 Core.

If I try

```
ha supervisor upgrade --version 2023.12.0 (or ha supervisor downgrade --version 2023.12.0)
```
i see
```
Error: No supervisor update available - 2024.01.1
```

**Comment by Stefano201** [comment]
UP
